js/bytex.js

```diff
@@ -0,0 +1,1455 @@
+                'www': 'https://bytex.ca',
+                'doc': 'https://docs.bytex.ca',
+                'referral': 'https://bytex.ca/signup?affiliation_code=EREEF6',
+            },
+            'precisionMode': TICK_SIZE,
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': true,
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'health',
+                        'constants',
+                        'kit',
+                        'tiers',
+                        'ticker',
+                        'tickers',
+                        'orderbook',
+                        'orderbooks',
+                        'trades',
+                        'chart',
+                        'charts',
+                        // TradingView
+                        'udf/config',
+                        'udf/history',
+                        'udf/symbols',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'user',
+                        'user/balance',
+                        'user/deposits',
+                        'user/withdrawals',
+                        'user/withdrawal/fee',
+                        'user/trades',
+                        'orders',
+                        'orders/{order_id}',
+                    ],
+                    'post': [
+                        'user/request-withdrawal',
+                        'order',
+                    ],
+                    'delete': [
+                        'order/all',
+                        'order',
+                    ],
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': true,
+                    'percentage': true,
+                    'taker': 0.001,
+                    'maker': 0.001,
+                },
+            },
+            'exceptions': {
+                'broad': {
+                    'Invalid token': AuthenticationError,
+                    'Order not found': OrderNotFound,
+                    'Insufficient balance': InsufficientFunds,
+                },
+                'exact': {
+                    '400': BadRequest,
+                    '403': AuthenticationError,
+                    '404': BadRequest,
+                    '405': BadRequest,
+                    '410': BadRequest,
+                    '429': BadRequest,
+                    '500': NetworkError,
+                    '503': NetworkError,
+                },
+            },
+            'options': {
+                // how many seconds before the authenticated request expires
+                'api-expires': parseInt (this.timeout / 1000),
+                'networks': {
+                    'TRC20': 'trx',
+                    'ERC20': 'eth',
+                    'trx': 'trx',
+                    'eth': 'eth',
```

Comment:
@ByteX-Dev Do you think you could update `options['networks']` within describe and `safeNetwork` with all the networks that are on Bytex? You can check some exchange files like Binance, Gateio and FTX for the unified network codes. 

Thanks!

Reply:
Absolutely!